index.js

```diff
@@ -0,0 +1,5 @@
+const server = require('./api/server')
```

Comment:
you dont have an api folder, your server file is in the root directory. can you fix this import to get your app in a working state, thank you